services/provider/server/server.go

```diff
@@ -1229,13 +1230,39 @@ func (s *OCSProviderServer) getDesiredCephConnection(
 		return nil, fmt.Errorf("failed to extract monitor IPs from configmap %s: %v", monConfigMap, err)
 	}
 
+	var readAffinity *csiopv1a1.ReadAffinitySpec = nil
```

Comment:
Please move the definition of the var the nearest place above the place where it is being used

Reply:
done